lua/dap.lua

```diff
@@ -57,6 +57,9 @@ M.defaults = setmetatable(
   {
     fallback = {
       exception_breakpoints = 'default';
+      ask_step_in_targets = true;
+      -- type SteppingGranularity = 'statement' | 'line' | 'instruction'
+      stepping_granularity = 'statement';
```

Comment:
Also added SteppingGranularity. This does not require a capability but most adapters will ignore this information.

Reply:
actually all step request support granularity

Reply:
except "continue"/"reverseContinue"